lib/beaker/gauge.ex

```diff
@@ -132,6 +132,32 @@ defmodule Beaker.Gauge do
   end
 
   @doc """
+  Sets gauge with min max values
+
+  ## Parameters
+    * `key`: The name of the gauges to set the value for.
+    * `value`: The value to set for the gauge
+    * `min`: The lowest value for the gauge
+    * `max`: The highest value for the gauge
+    * `auto_update`: Update min max based on received values, defaults to `false`
+
+
+  ## Examples
+
+      iex> Beaker.Gauge.set("latency", 70, 0, 1000)
+      :ok
+      iex> Beaker.Gauge.get("latency")
+      %{name: "latency", value: 70, min: 0, max: 1000}
+
+
+   """
+
+  def set(key, value, min, max) do
```

Comment:
Are we planning to do anything to enforce the min/max of the gauge?
Also, what about a default of `nil` which would act as infinity for times when a limit is not cared for?